client/src/components/AppHeader/AppHeader.js

```diff
@@ -1,48 +1,42 @@
 import logo from './logo.png';
 import React from 'react';
-import './style.scss'
-import { Navbar,  FormControl, Form, Nav, InputGroup, Button } from 'react-bootstrap';
+import './AppHeader.scss'
+import { Navbar,  FormControl, Form, Nav, InputGroup } from 'react-bootstrap';
 import { faSignInAlt } from '@fortawesome/free-solid-svg-icons'
 import { faUsers } from '@fortawesome/free-solid-svg-icons'
 import { faCalendarAlt } from '@fortawesome/free-solid-svg-icons'
-import { faSearch } from '@fortawesome/free-solid-svg-icons'
 import { FontAwesomeIcon } from '@fortawesome/react-fontawesome'
 
 const AppHeader = props => {
```

Comment:
You can shorten this syntax const AppHeader = props => {return ()} to mo simple variant
AppHeader = props => ()

Reply:
Later we'll create state in this component, so I think we can pretermit this...

Reply:
Okay, thank you!